packages/api-client/src/store/store.ts

```diff
@@ -290,8 +290,14 @@ export const createWorkspaceStore = (
   // PROXY URL STATE
   const PROXY_URL = 'proxyUrl' as const
 
-  // Initialize proxyUrl with the value from localStorage
-  const proxyUrl = ref(localStorage.getItem(PROXY_URL) || '')
```

Comment:
Desperately needed, but I don’t see a `proxyUrl` to be stored in local storage. Is that really where we get it from? Because I can’t see it using the browser console. In the past it has been stored in the workspace.

Reply:
lets get this in but re-assess this later in a FF

right now its hard to use https://client.scalar.com without any form of error reporting that cors is disabled.

ill make an issue

Reply:
https://github.com/scalar/scalar/issues/3541